strongbox-db-import/src/main/java/org/carlspring/strongbox/db/orient/OrientDbConfiguration.java

```diff
@@ -1,53 +1,77 @@
 package org.carlspring.strongbox.db.orient;
 
-import javax.sql.DataSource;
 import java.lang.reflect.Field;
 import java.util.Properties;
 
-import com.orientechnologies.orient.core.db.ODatabasePool;
-import com.orientechnologies.orient.core.db.ODatabaseType;
-import com.orientechnologies.orient.core.db.OrientDB;
-import com.orientechnologies.orient.core.db.OrientDBConfig;
-import com.orientechnologies.orient.jdbc.OrientDataSource;
-import lombok.extern.slf4j.Slf4j;
+import javax.sql.DataSource;
+
 import org.apache.commons.lang3.StringUtils;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.boot.autoconfigure.liquibase.LiquibaseDataSource;
+import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.DependsOn;
 import org.springframework.util.ReflectionUtils;
+import org.strongbox.db.server.EmbeddedOrientDbServer;
+import org.strongbox.db.server.OrientDbServer;
+import org.strongbox.db.server.OrientDbServerConfiguration;
+import org.strongbox.db.server.OrientDbServerProperties;
+import org.strongbox.db.server.OrientDbStudioConfiguration;
+import org.strongbox.db.server.OrientDbStudioProperties;
+
+import com.orientechnologies.orient.core.db.ODatabasePool;
+import com.orientechnologies.orient.core.db.ODatabaseType;
+import com.orientechnologies.orient.core.db.OrientDB;
+import com.orientechnologies.orient.core.db.OrientDBConfig;
+import com.orientechnologies.orient.jdbc.OrientDataSource;
 
 /**
  * @author Przemyslaw Fusik
  */
-@Slf4j
 @Configuration
 class OrientDbConfiguration
 {
+    
+    private static final Logger logger = LoggerFactory.getLogger(OrientDbConfiguration.class);
+    
+    @Bean
+    OrientDbServer orientDbServer(OrientDbServerConfiguration serverProperties, OrientDbStudioConfiguration studioProperties) {
```

Comment:
Cosmetic: Braces on next line, please! (Here and below and in various other places across the pull request).